src/dune_rpc_impl/dune

```diff
@@ -1,16 +1,18 @@
 (library
  (name dune_rpc_impl)
  (libraries
-  stdune
-  promote
-  unix
-  fiber
   csexp_rpc
-  dune_stats
-  dune_rpc_client
   dune_console
-  dune_util
+  dune_engine
+  dune_lang
+  dune_rpc_client
   dune_rpc_private
   dune_rpc_server
-  dune_engine)
+  dune_stats
+  dune_util
+  fiber
+  promote
+  stdune
+  unix)
  (synopsis "Dune's rpc server + a usable client"))
+
```

Comment:
Pet peeve, but this should be sorted. I added dune_lang as I pull it up as dependency in server.ml